client/matchmaking/browse-map-selections.jsx

```diff
@@ -256,22 +254,10 @@ export default class MapSelections extends React.Component {
     const { auth, matchmaking, matchmakingPreferences, type } = this.props
 
     const mapPool = matchmaking.mapPoolTypes.get(type)
-    const preferencesRecord = matchmakingPreferences.typeToPreferences.get(type)
+    const prefs = matchmakingPreferences.typeToPreferences.get(type)
```

Comment:
Just a note, no need to fix in this PR, but when we update this code in the future to move to hooks, we probably want to select more granular things in the `useAppSelector` call. e.g. it's better to return this preference record from the `useAppSelector` call instead of returning the entire state object, since it will prevent re-renders when state in non-relevant parts of the state changes.